src/components/icons/template.tsx

```diff
@@ -0,0 +1,28 @@
+// import * as React from 'react'
+import styled from 'styled-components'
+
+export interface IconProps {
+  // text description, for accessiblity
+  title?: string
+}
+
+export default function GenerateIconComponentForGraphic(Graphic: JSX.Element): any {
```

Comment:
Can we have the same name as you use it in other files `GenerateComponentForGraphic`